tardis/model/base.py

```diff
@@ -308,12 +299,17 @@ def __init__(
             )
             t_radiative = constants.b_wien / (
                 lambda_wien_inner
-                * (1 + (self.v_middle - self.v_boundary_inner) / constants.c)
+                * (
+                    1
+                    + (self.v_middle - self.geometry.v_inner_boundary)
+                    / constants.c
+                )
             )
-        elif len(t_radiative) != self.no_of_shells:
+        
+        elif len(t_radiative) == self.no_of_shells + 1:
             t_radiative = t_radiative[
-                self.v_boundary_inner_index
-                + 1 : self.v_boundary_outer_index
+                self.geometry.v_inner_boundary_index
```

Comment:
Change v_inner_boundary_index to inner_boundary_index

Reply:
can we make that an issue for now?